deepchecks/vision/utils/detection_encoders.py

```diff
@@ -0,0 +1,64 @@
+from typing import Union, Callable
+
+__all__ = ["DetectionLabelEncoder", "DetectionPredictionEncoder"]
+
+
+class DetectionLabelEncoder:
+    """
+    Class for encoding the detection annotations to the required format.
+
+    Parameters
+    ----------
+    label_encoder : Union[str, Callable]
+        Function or string that specifies the encoding function.
+        If a string, it must be one of the following:
```

Comment:
State that in all these, class comes before coordinates.

Reply:
Also need to note that the batch label is a list containing tensors with axis=0 being the axis of bounding boxes and axis=1 being the specified format.